nixos/modules/services/networking/nftables.nix

```diff
@@ -70,6 +70,17 @@ in
       '';
     };
 
+    networking.nftables.checkRulesetFlatten = mkOption {
+      type = types.bool;
+      default = false;
+      description = lib.mdDoc ''
+        Flatten the ruleset file before checking it. This is useful if the ruleset
+        file includes other files that are read-only in the sandbox environment.
+
+        This option will use builtins.readFile instead of include before checking.
+      '';
+    };
```

Comment:
Users reading the name and description of this option may expect that the substitution only happens at check time, leaving `include` unchanged in the final ruleset.  However, it not the case.  Could you improve the wording a bit?

Reply:
does this break any existing usecase? without this rulesetFile is partially broken so we could just always use this

Reply:
ah sorry didnt see the above comment

Reply:
> we could just always use this

Yes, another implementation choice is always doing so if `cfg.rulesetFile` is a store path.

Reply:
> Could you improve the wording a bit?

What about this change?